.github/workflows/charmcraft-pack.yaml

```diff
@@ -11,6 +11,7 @@ jobs:
       uses: actions/checkout@v3
       with:
         repository: jnsgruk/hello-kubecon
+        ref: dbd133466dde59ee64f20a732a8f3d2e560ec3b8  # master, July 3rd, 2023
```

Comment:
I don't think we need to have the branch name in the comment, since we'll always be using whatever the default branch is - it also avoids adding "master" into our repo. I do like the date stamp, though - but if we do that then we should do it for all of them for consistency.

Reply:
Reworked.